sorc/compile_evs_global_ens_sorc.sh

```diff
@@ -4,7 +4,7 @@ SORCevs_gens=`pwd`
 
 source ../versions/build.ver
 
-source ../modulefiles/v1.0
+source ../modulefiles/v1.0.old
```

Comment:
This file does not exist in this feature branch. Let's remove the .old extension.